.github/workflows/test.yml

```
@@ -0,0 +1,19 @@
+name: Test
+on: [push]
```

Same feedback as https://github.com/forcedotcom/sf-fx-runtime-nodejs/pull/436#discussion_r1012065172

```suggestion
on:
  push:
    # Avoid duplicate builds on PRs.
    branches:
      - main
  pull_request:
permissions:
  contents: read
```